external/include/mbedtls/see_api.h

```diff
@@ -45,6 +45,11 @@
 #include "../../../os/arch/arm/src/s5j/sss/isp_driver_error.h"
 #include "../../../os/arch/arm/src/s5j/sss/isp_oid.h"
 
+#include "../../../os/arch/arm/src/s5j/sss/isp_driver_aes_securekey.h"
```

Comment:
1. Why do you add this in here? The mbedtls is a common module, not a architecture specific module.
2. If you want to include these, you had better add that path at makefile first.